gwtp-core/gwtp-mvp-client/src/main/resources/com/gwtplatform/mvp/processors/ProxyPlace.vm

```diff
@@ -0,0 +1,49 @@
+package $impl.packageName;
+
+import javax.annotation.Generated;
+import javax.inject.Inject;
+import javax.inject.Provider;
+import com.google.web.bindery.event.shared.EventBus;
+import com.gwtplatform.common.client.StandardProvider;
+import com.gwtplatform.mvp.client.proxy.PlaceImpl;
+import com.gwtplatform.mvp.client.proxy.PlaceManager;
+import com.gwtplatform.mvp.client.proxy.ProxyImpl;
+import com.gwtplatform.mvp.client.proxy.ProxyPlaceImpl;
+import com.gwtplatform.mvp.client.proxy.RevealContentHandler;
+#printImports($imports)
+
+@Generated(value = "$processor.qualifiedName", date = "$processingDate")
+public class $impl.simpleName extends ProxyPlaceImpl<$presenterType.simpleName> implements $proxyType.simpleName {
+## TODO: This is a basically a SimpleProxy with a hard-coded name. Reuse the other file
```

Comment:
Ah right. I recall that at some point we talked about removing the WrappedProxy?